### ISSUE TITLE
GitHub Actions version pinning dropped

### ISSUE THREAD

**Issue by na-jakobs** [dependabot/dependabot-core#7971]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Package ecosystem

github-actions

### dependabot.yml content

```
version: 2
updates:
  - package-ecosystem: "github-actions"
    directory: "/"
    schedule:
      interval: "daily"
```

### What you expected to see, versus what you actually saw

Expected to see an update to the latest sha along with an update to the comment.

Removed:
```
steps:
   - name: Checkout repo
     uses: actions/checkout@8e5e7e5ab8b370d6c329ec480221332ada57f0ab # v3.5.2
```

Added:
```
steps:
   - name: Checkout repo
     uses: actions/checkout@v4 # v3.5.2
```

### Images of the diff or a link to the PR, issue, or logs

![dependabot-removing-sha](https://github.com/dependabot/dependabot-core/assets/64911732/871fe484-e6d9-4905-9c87-5f669265837d)

**Comment by RazerM** [comment]
I have the same issue in https://github.com/getlogbook/logbook/pull/356/files

**Comment by yeikel** [comment]
> I have the same issue in https://github.com/getlogbook/logbook/pull/356/files

Your issue seems to be a valid duplicate of this https://github.com/dependabot/dependabot-core/issues/7966

In particular, the reason seems to be this : https://github.com/dependabot/dependabot-core/issues/7966#issuecomment-1706057699

@na-jakobs Can you share a link to your pull request/manifest? I am trying to see if the symptoms are the same as the pull request shared above. If so, we can probably close this as duplicate and upvote the other

**Comment by RazerM** [comment]
@yeikel I'm referring to this file, which dependabot is bumping a commit hash to a tag: https://github.com/getlogbook/logbook/pull/356/files#diff-ab7cd36ad831c30099773459b80595fd3b72612f333f6a2f4e11c2606176999f

**Comment by deivid-rodriguez** [comment]
This will get fixed by #8068!
